cookbook/deployment/fortrabbit.rst

```diff
@@ -0,0 +1,261 @@
+            'handlers' => array(
+                'nested' => array(
+                    'type' => 'error_log',
+                ),
+            ),
+        ));
+
+Configuring Database Access & Session Handler
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can use the fortrabbit App Secrets to attain your database credentials. 
+Create the file ``app/config_prod_secrets.php`` with the following contents::
+
+    <?php
+    // Get the path to the secrects.json file
+    if (!$secrets = getenv("APP_SECRETS")) {
+        return;
+    }
+
+    // Read the file and decode json to an array
+    $secrets = json_decode(file_get_contents($secrets), true);
+
+    // Set database parameters to the container
+    if (isset($secrets['MYSQL'])) {
+
+        $container->setParameter('database_driver', 'pdo_mysql');
+        $container->setParameter('database_host', $secrets['MYSQL']['HOST']);
+        $container->setParameter('database_name', $secrets['MYSQL']['DATABASE']);
+        $container->setParameter('database_user', $secrets['MYSQL']['USER']);
+        $container->setParameter('database_password', $secrets['MYSQL']['PASSWORD']);
+    }
+
+    // Check if the Memcache component is present
+    if (isset($secrets['MEMCACHE'])) {
+
+        $memcache = $secrets['MEMCACHE'];
+        $handlers = [];
+
+        foreach (range(1, $memcache['COUNT']) as $num) {
+            $handlers [] = $memcache['HOST' . $num] . ':' . $memcache['PORT' . $num];
+        }
+
+        // Apply ini settings
+        ini_set('session.save_handler', 'memcached');
+        ini_set('session.save_path', implode(',', $handlers));
+
+        if ($memcache['COUNT'] == 2) {
+            ini_set('memcached.sess_number_of_replicas', 1);
+            ini_set('memcached.sess_consistent_hash', 1);
+            ini_set('memcached.sess_binary', 1);
+        }
+    }
+
+Make sure this file is listed in your *imports*:
+
+.. code-block:: yaml
+
+    # app/config/config_prod.yml
+    imports:
+        - { resource: config.yml }
+        - { resource: config_prod_secrets.php }
+
+    # ..
+    framework:
+        session:
+            # set handler_id to null to use default session handler from php.ini (memcached)
+            handler_id:  ~
+    # ..
+
+
+Configuring the Environment in the Dashboard
+--------------------------------------------
+
+PHP Settings
+~~~~~~~~~~~~
+
+The PHP version and enabled extensions are configuable under the PHP settings 
+of your App within the fortrabbit Dashboard.
+
+
+Environment Variables
+~~~~~~~~~~~~~~~~~~~~~
+
+Set the ``SYMFONY_ENV`` environment variable to ``prod`` to make sure the right 
+config files get loaded. ENV vars are configuable in fortrabbit Dashboard as well.
+
+
+Document Root
+~~~~~~~~~~~~~
+
+The document root is configuable for every custom domain you setup for your App. 
+The default is /htdocs, but for Symfony you probably want to change it to 
+/htdocs/web. You also do so in the fortrabbit Dashboard under ``Domain`` settings.
```

Comment:
Paths here should be enclosed with double backticks (can you check the rest of the document too?).